src/AppInstallerCommonCore/Public/AppInstallerTelemetry.h

```diff
@@ -147,6 +161,9 @@ namespace AppInstaller::Logging
         DestructionToken m_token;
     };
 
+    // Turns on wil failure telemetry and logging.
+    void EnableWilFailureTelemetry();

```

Comment:
Will revert the location of this method's declaration to reflect no change to this method!